keepassxc-browser/options/options.html

```diff
@@ -2,439 +2,1034 @@
 <html>
   <head>
     <title data-i18n="optionsTitle"></title>
-      <meta charset="UTF-8">
-      <link rel="stylesheet" href="bootstrap.min.css" />
-      <link rel="stylesheet" href="options.css" />
-      <script src="../browser-polyfill.min.js"></script>
-      <script src="../global.js"></script>
-      <script src="../jquery-3.3.1.min.js"></script>
-      <script src="bootstrap.min.js"></script>
-      <script src="options.js"></script>
-      <script src="../translate.js" defer></script>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="/styles/bootstrap.css" />
+    <link rel="stylesheet" href="options.css" />
+    <link rel="stylesheet" href="../fonts/fork-awesome.min.css" />
+    <script type="text/javascript" src="../browser-polyfill.min.js"></script>
+    <script type="text/javascript" src="../global.js"></script>
+    <script type="text/javascript" src="../jquery-3.3.1.min.js"></script>
+    <script type="text/javascript" src="bootstrap.min.js"></script>
+    <script type="text/javascript" src="options.js"></script>
+    <script src="../translate.js" defer></script>
   </head>
+
   <body>
     <div class="container">
-      <h3 class="muted"><img src="/icons/keepassxc_48x48.png" alt="logo"> KeePassXC-Browser</h3>
-
-      <nav class="navbar navbar-default">
-        <ul class="nav navbar-nav">
-          <li class="active"><a href="#general-settings" tabindex="1"><span data-i18n="optionsMenuGeneral"></span></a></li>
-          <li><a href="#connected-databases" tabindex="2"><span data-i18n="optionsMenuConnectedDatabases"></span></a></li>
-          <li><a href="#custom-fields" tabindex="3"><span data-i18n="optionsMenuCustomFields"></span></a></li>
-          <li><a href="#site-preferences" tabindex="4"><span data-i18n="optionsMenuSitePreferences"></span></a></li>
-          <li><a href="#about" tabindex="5"><span data-i18n="optionsMenuAbout"></span></a></li>
-        </ul>
+      <h3><img src="/icons/keepassxc_48x48.png" alt="logo" /> KeePassXC-Browser</h3>
+
+      <nav class="navbar navbar-expand-lg navbar-dark bg-primary text-white mb-4">
+        <button
```

Comment:
Is there any reason for this file to be in different syntax than the rest of the HTML files? Why are the elements divided in multiple lines?